spec/system/index_spec.rb

```diff
@@ -74,9 +74,6 @@
       # in order to be completed by autocomplete
       fill_in 'Enter stop name', with: bus_stop.name.chars.first
       find('div', text: bus_stop.name).click
-      within 'form', text: 'Enter stop name' do
```

Comment:
Don't you still need to click the search button for this test to pass?

Reply:
After implementing the final behavior "the simpler option would be for the autocomplete to just redirect to the show routing using the ID in the autocomplete search string, if possible", it now just redirects when an option is clicked.

Reply:
I was playing with an older version. 😅 
So now the search button is unnecessary??

Reply:
Axe it, but on two conditions: Pressing enter should still work (for no-mouse users), and make sure to add some text so the user knows that they have to select a stop to go to it.
I'm not sure removing the search is good from a UI perspective, we'll have to see.

Reply:
Revisiting this, on second look I don't think we should remove the button. A user could theoretically type something in, then click elsewhere on the page to dismiss the autocomplete. At this point if we remove the button and the controller logic, they can't do anything. If we just remove the button, they still will probably have no idea what to do and even if they figure out that they should re-click on the field and press enter, they would still invoking the same controller logic that the button was previously linked to since the hastus ids is only available from within the autocomplete.

Reply:
Ahh good thinking. Agreed.